### ISSUE TITLE
racketblock always prints decimal numbers 

### ISSUE THREAD

**Issue by andmkent** [racket/scribble#24]
```racketblock``` throws away the original format of numbers in code, replacing them with their equivalent decimal representation.

For instance, this example produces a document with ```255``` instead of ```#xff```:

```racket
#lang scribble/base
@(require scribble/manual)

@racketblock[#xff]
```

**Comment by mflatt** [comment]
Yes, that's a limitation of `racketblock`. The `code` form uses a different approach that preserves the original format of a number.

**Comment by samth** [comment]
Could we fix this by using the same syntax property approach that's used for []?
